index.html

```diff
@@ -62,5 +62,223 @@ <h1>I'm Shah Zaman. Glad to see you!</h1>
+						</div>
+					</div>
+          <div class="card_discription">
+						<p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+					</div>
+          <div class="card_tags">
+						<div class="tag">
+							<button class="small">HTML</button>
+						</div>
+						<div class="tag">
+							<button class="small">CSS</button>
+						</div>
+						<div class="tag">
+							<button class="small">Javascript</button>
+						</div>
+					</div>
+          <div class="card_action">
+						<button class="secondary">See Project</button>
+					</div>
+				</div>
+			</div>
+			<div class="card_works">
+				<img src="./images/tonic.svg" alt="SnapshootPortfolio">
+				<div class="card_detail">
+          <div class="card_title">
+						<h2>Tonic</h2>
+						<div class="card_history">
+               <div class="card_history_detail"><p>CANOPY</p></div>
+               <div><img src="./images/Counter.svg" alt=""></div>
+               <div class="card_history_detail"><p>Back End Dev</p></div>
+               <div><img src="./images/Counter.svg" alt=""></div>
+               <div class="card_history_detail"><p>2015</p></div>
+						</div>
+					</div>
+          <div class="card_discription">
+						<p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+					</div>
+          <div class="card_tags">
+						<div class="tag">
+							<button class="small">HTML</button>
+						</div>
+						<div class="tag">
+							<button class="small">CSS</button>
+						</div>
+						<div class="tag">
+							<button class="small">Javascript</button>
+						</div>
+					</div>
+          <div class="card_action">
+						<button class="secondary">See Project</button>
+					</div>
+				</div>
+			</div>
+			<div class="card_works">
+				<img src="./images/multiPostStoriess.svg" alt="SnapshootPortfolio">
+				<div class="card_detail">
+          <div class="card_title">
+						<h2>Multi-Post
+							Stories</h2>
+						<div class="card_history">
+               <div class="card_history_detail"><p>CANOPY</p></div>
+               <div><img src="./images/Counter.svg" alt=""></div>
+               <div class="card_history_detail"><p>Back End Dev</p></div>
+               <div><img src="./images/Counter.svg" alt=""></div>
+               <div class="card_history_detail"><p>2015</p></div>
+						</div>
+					</div>
+          <div class="card_discription">
+						<p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+					</div>
+          <div class="card_tags">
+						<div class="tag">
+							<button class="small">HTML</button>
+						</div>
+						<div class="tag">
+							<button class="small">CSS</button>
+						</div>
+						<div class="tag">
+							<button class="small">Javascript</button>
+						</div>
+					</div>
+          <div class="card_action">
+						<button class="secondary">See Project</button>
```

Comment:
- [ ] Kindly add the following functions to your button in the project. Active, Hover, and Pressed to match with the design provided

![image](https://github.com/sheezu900/My_Potfolio/assets/93630700/f297bae2-0205-43ad-aeba-6e03e511cf25)